test/JDBC/upgrade/16_4/schedule

```diff
@@ -556,7 +556,7 @@ BABEL-3820
 stuff
 PARTITION
 BABEL-5071
-string_agg
+string_agg_within
```

Comment:
we should add `string_agg-before-15_9-or-16_5` and  `string_agg_within-16_4` instead of `string_agg_within`

Reply:
done